bokeh/sphinxext/bokehjs_content.py

```diff
@@ -232,8 +172,7 @@ def run(self):
             source_doc.bjs_seen = True
             node['include_bjs_header'] = True
         code_content, language = self.get_code_language()
-        cb = self.get_codeblock_node(
-            code_content, language)
+        cb = get_codeblock_node(self, code_content, language)
```

Comment:
Decided to move this method to utils.py since it's needed in bokehjs_script.py

Reply:
Technically would be a breaking change but the transitive import makes it actually not, I suppose. I am fine to move it in any case since to be honest `bokeh.sphinxext` is really something slightly separate from the main library (also I can't imagine any reason anyone would be using this function)

Reply:
By the way, was this method copied from `sphinx.directives.code.CodeBlock.run` because of the way codeblocks handle HTML? I noticed there were some slight differences between the two methods. Couldn't get the HTML to work with the `run()` method - it loaded characters line by line for me 😂

Reply:
probably, but `bokehjs_content` was written quite awhile ago and also in a huge hurry, I so I don't recall for certain